src/internal_modules/roc_audio/EndToEndLatencyMonitor.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 Roc Streaming authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//
+// Created by misha on 31.08.23.
+//
+
+#include "EndToEndLatencyMonitor.h"
+
+namespace roc {
+namespace audio {
+
+EndToEndLatencyMonitor::EndToEndLatencyMonitor(IFrameReader& reader)
```

Comment:
I suggest to merge this functionality into LatencyMonitor.

Later LatencyMonitor will anyway need it when we will implement clock sync mode based on RTCP. So there is no need to have two monitors.